android/build.gradle

```diff
@@ -1,21 +1,56 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext {
+        buildToolsVersion = "28.0.3"
+        minSdkVersion = 16
+        compileSdkVersion = 28
+        targetSdkVersion = 28
+        supportLibVersion = "28.0.0"
+    }
     repositories {
         flatDir { dirs "libs", "${rootDir}/app/libs" }
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
-        classpath 'com.google.gms:google-services:4.1.0'
+        classpath 'com.android.tools.build:gradle:3.3.2'
+        classpath 'com.google.gms:google-services:4.0.1'
```

Comment:
Downgraded?

Reply:
fixed

Reply:
added comment to code, it actually fixes one problem https://github.com/invertase/react-native-firebase/issues/766#issuecomment-365116012